### PR TITLE
Pull Request: added 2 new keymaps to DZ60 keyboard

### PR DESCRIPTION
**Pull request by spotpuff**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/dz60/keymaps/coppertop/keymap.c [hunk 1]
```diff
@@ -0,0 +1,80 @@
+#include QMK_KEYBOARD_H
+
+//Layers
+enum layer_names {
+	_QWERTY,
+	_FN,
+	_MEDIA,
```
Comment (drashna) [inline keyboards/dz60/keymaps/coppertop/keymap.c:7]:
```suggestion
    _QWERTY,
    _FN,
    _MEDIA,
```

File: keyboards/dz60/keymaps/coppertop/keymap.c [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+ *  `----------------------------------------------------------------------------------------'
+ */
+
+	[_QWERTY] = LAYOUT_directional(
+		KC_ESC,  KC_1,     KC_2,    KC_3,   KC_4,    KC_5,   KC_6,  KC_7,   KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC, KC_DEL,
+		KC_TAB,  KC_Q,     KC_W,    KC_E,   KC_R,    KC_T,   KC_Y,  KC_U,   KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,
+		LT(_MEDIA,KC_GRV), KC_A,    KC_S,   KC_D,    KC_F,   KC_G,  KC_H,   KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_ENT,
+		KC_LSFT, KC_Z,     KC_X,    KC_C,   KC_V,    KC_B,   KC_N,  KC_M,   KC_COMM, KC_DOT,  KC_SLSH, KC_HOME, KC_UP,   KC_END,
+		KC_LCTL, KC_LGUI,  KC_LALT, KC_SPC, MO(_FN), KC_SPC, KC_F7, KC_F10, KC_LEFT, KC_DOWN, KC_RGHT
+		),
```
Comment (drashna) [inline keyboards/dz60/keymaps/coppertop/keymap.c:35]:
```suggestion
    [_QWERTY] = LAYOUT_directional(
        KC_ESC,  KC_1,     KC_2,    KC_3,   KC_4,    KC_5,   KC_6,  KC_7,   KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC, KC_DEL,
        KC_TAB,  KC_Q,     KC_W,    KC_E,   KC_R,    KC_T,   KC_Y,  KC_U,   KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,
        LT(_MEDIA,KC_GRV), KC_A,    KC_S,   KC_D,    KC_F,   KC_G,  KC_H,   KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_ENT,
        KC_LSFT, KC_Z,     KC_X,    KC_C,   KC_V,    KC_B,   KC_N,  KC_M,   KC_COMM, KC_DOT,  KC_SLSH, KC_HOME, KC_UP,   KC_END,
        KC_LCTL, KC_LGUI,  KC_LALT, KC_SPC, MO(_FN), KC_SPC, KC_F7, KC_F10, KC_LEFT, KC_DOWN, KC_RGHT
        ),
```

File: keyboards/dz60/keymaps/coppertop/keymap.c [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+ *  `----------------------------------------------------------------------------------------'
+ */
+
+	[_FN] = LAYOUT_directional(
+		XXXXXXX, KC_F1,   KC_F2,   KC_F3,   KC_F4,    KC_F5,    KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  XXXXXXX, KC_INS,
+		XXXXXXX, RGB_TOG, RGB_HUI, RGB_SAI, RGB_VAI,  XXXXXXX,  XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, KC_CAD,
+		KC_CAPS, XXXXXXX, RGB_HUD, RGB_SAD, RGB_VAD,  XXXXXXX,  XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, KC_SLCK, KC_PAUS, KC_PSCR,
+		_______, RGB_M_P, RGB_M_B, RGB_M_R, RGB_M_SW, RGB_M_SN, RGB_M_K, RGB_M_X, RGB_M_G, XXXXXXX, XXXXXXX, KC_PGUP, BL_INC,  KC_PGDN,
+		_______, _______, _______, XXXXXXX, _______,  XXXXXXX,  XXXXXXX, XXXXXXX, BL_TOGG, BL_DEC,  BL_STEP
+		),
```
Comment (drashna) [inline keyboards/dz60/keymaps/coppertop/keymap.c:57]:
```suggestion
    [_FN] = LAYOUT_directional(
        XXXXXXX, KC_F1,   KC_F2,   KC_F3,   KC_F4,    KC_F5,    KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  XXXXXXX, KC_INS,
        XXXXXXX, RGB_TOG, RGB_HUI, RGB_SAI, RGB_VAI,  XXXXXXX,  XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, KC_CAD,
        KC_CAPS, XXXXXXX, RGB_HUD, RGB_SAD, RGB_VAD,  XXXXXXX,  XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, KC_SLCK, KC_PAUS, KC_PSCR,
        _______, RGB_M_P, RGB_M_B, RGB_M_R, RGB_M_SW, RGB_M_SN, RGB_M_K, RGB_M_X, RGB_M_G, XXXXXXX, XXXXXXX, KC_PGUP, BL_INC,  KC_PGDN,
        _______, _______, _______, XXXXXXX, _______,  XXXXXXX,  XXXXXXX, XXXXXXX, BL_TOGG, BL_DEC,  BL_STEP
        ),
```

File: keyboards/dz60/keymaps/coppertop/keymap.c [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+ */
+
+	[_MEDIA] = LAYOUT_directional(
+		XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, KC_PSLS, KC_PAST, KC_PMNS, KC_PEQL, XXXXXXX, _______, _______, 
+		XXXXXXX, XXXXXXX, KC_MPRV, KC_MPLY, KC_MNXT, XXXXXXX, XXXXXXX, KC_P7,   KC_P8,   KC_P9,   KC_PPLS, XXXXXXX, XXXXXXX, RESET, 
+		_______, XXXXXXX, KC_MUTE, KC_VOLD, KC_VOLU, XXXXXXX, XXXXXXX, KC_P4,   KC_P5,   KC_P6,   KC_PPLS, XXXXXXX, _______, 
+		_______, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, KC_P1,   KC_P2,   KC_P3,   KC_PENT, XXXXXXX, XXXXXXX, XXXXXXX, 
+		_______, _______, _______, _______, _______, KC_P0,   KC_PDOT, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX
+		)
+};
```
Comment (drashna) [inline keyboards/dz60/keymaps/coppertop/keymap.c:80]:
```suggestion
    [_MEDIA] = LAYOUT_directional(
        XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, KC_PSLS, KC_PAST, KC_PMNS, KC_PEQL, XXXXXXX, _______, _______, 
        XXXXXXX, XXXXXXX, KC_MPRV, KC_MPLY, KC_MNXT, XXXXXXX, XXXXXXX, KC_P7,   KC_P8,   KC_P9,   KC_PPLS, XXXXXXX, XXXXXXX, RESET, 
        _______, XXXXXXX, KC_MUTE, KC_VOLD, KC_VOLU, XXXXXXX, XXXXXXX, KC_P4,   KC_P5,   KC_P6,   KC_PPLS, XXXXXXX, _______, 
        _______, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, KC_P1,   KC_P2,   KC_P3,   KC_PENT, XXXXXXX, XXXXXXX, XXXXXXX, 
        _______, _______, _______, _______, _______, KC_P0,   KC_PDOT, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX
        )
};
```

File: keyboards/dz60/keymaps/spotpuff/keymap.c [hunk 5]
```diff
@@ -0,0 +1,80 @@
+#include QMK_KEYBOARD_H
+
+//Layers
+enum layer_names {
+	_QWERTY,
+	_FN,
+	_MEDIA,
```
Comment (drashna) [inline keyboards/dz60/keymaps/spotpuff/keymap.c:7]:
```suggestion
    _QWERTY,
    _FN,
    _MEDIA,
```

File: keyboards/dz60/keymaps/spotpuff/keymap.c [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+ *  `----------------------------------------------------------------------------------------'
+ */
+
+	[_QWERTY] = LAYOUT_directional(
+		KC_ESC,  KC_1,     KC_2,    KC_3,   KC_4,    KC_5,   KC_6,    KC_7,   KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC, KC_DEL, 
+		KC_TAB,  KC_Q,     KC_W,    KC_E,   KC_R,    KC_T,   KC_Y,    KC_U,   KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS, 
+		LT(_MEDIA,KC_GRV), KC_A,    KC_S,   KC_D,    KC_F,   KC_G,    KC_H,   KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_ENT, 
+		KC_LSFT, KC_Z,     KC_X,    KC_C,   KC_V,    KC_B,   KC_N,    KC_M,   KC_COMM, KC_DOT,  KC_SLSH, KC_PGUP, KC_UP,   KC_PGDN,
+		KC_LCTL, KC_LGUI,  KC_LALT, KC_SPC, MO(_FN), KC_SPC, KC_HOME, KC_END, KC_LEFT, KC_DOWN, KC_RGHT 
+		),
```
Comment (drashna) [inline keyboards/dz60/keymaps/spotpuff/keymap.c:35]:
```suggestion
    [_QWERTY] = LAYOUT_directional(
        KC_ESC,  KC_1,     KC_2,    KC_3,   KC_4,    KC_5,   KC_6,    KC_7,   KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC, KC_DEL, 
        KC_TAB,  KC_Q,     KC_W,    KC_E,   KC_R,    KC_T,   KC_Y,    KC_U,   KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS, 
        LT(_MEDIA,KC_GRV), KC_A,    KC_S,   KC_D,    KC_F,   KC_G,    KC_H,   KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_ENT, 
        KC_LSFT, KC_Z,     KC_X,    KC_C,   KC_V,    KC_B,   KC_N,    KC_M,   KC_COMM, KC_DOT,  KC_SLSH, KC_PGUP, KC_UP,   KC_PGDN,
        KC_LCTL, KC_LGUI,  KC_LALT, KC_SPC, MO(_FN), KC_SPC, KC_HOME, KC_END, KC_LEFT, KC_DOWN, KC_RGHT 
        ),
```

File: keyboards/dz60/keymaps/spotpuff/keymap.c [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+ *  `----------------------------------------------------------------------------------------'
+ */
+
+	[_FN] = LAYOUT_directional( 
+		XXXXXXX, KC_F1,   KC_F2,   KC_F3,   KC_F4,    KC_F5,    KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  XXXXXXX, KC_INS,
+		XXXXXXX, RGB_TOG, RGB_HUI, RGB_SAI, RGB_VAI,  XXXXXXX,  XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, KC_CAD,
+		KC_CAPS, XXXXXXX, RGB_HUD, RGB_SAD, RGB_VAD,  XXXXXXX,  XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, KC_SLCK, KC_PAUS, KC_PSCR,
+		_______, RGB_M_P, RGB_M_B, RGB_M_R, RGB_M_SW, RGB_M_SN, RGB_M_K, RGB_M_X, RGB_M_G, XXXXXXX, XXXXXXX, XXXXXXX, BL_INC,  XXXXXXX,
+		_______, _______, _______, XXXXXXX, _______,  XXXXXXX,  XXXXXXX, XXXXXXX, BL_TOGG, BL_DEC,  BL_STEP
+		),
```
Comment (drashna) [inline keyboards/dz60/keymaps/spotpuff/keymap.c:57]:
```suggestion
    [_FN] = LAYOUT_directional( 
        XXXXXXX, KC_F1,   KC_F2,   KC_F3,   KC_F4,    KC_F5,    KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  XXXXXXX, KC_INS,
        XXXXXXX, RGB_TOG, RGB_HUI, RGB_SAI, RGB_VAI,  XXXXXXX,  XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, KC_CAD,
        KC_CAPS, XXXXXXX, RGB_HUD, RGB_SAD, RGB_VAD,  XXXXXXX,  XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, KC_SLCK, KC_PAUS, KC_PSCR,
        _______, RGB_M_P, RGB_M_B, RGB_M_R, RGB_M_SW, RGB_M_SN, RGB_M_K, RGB_M_X, RGB_M_G, XXXXXXX, XXXXXXX, XXXXXXX, BL_INC,  XXXXXXX,
        _______, _______, _______, XXXXXXX, _______,  XXXXXXX,  XXXXXXX, XXXXXXX, BL_TOGG, BL_DEC,  BL_STEP
        ),
```

File: keyboards/dz60/keymaps/spotpuff/keymap.c [hunk 8]
```diff
@@ -0,0 +1,80 @@
...
+ */
+
+  [_MEDIA] = LAYOUT_directional(
+		XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, KC_NLCK, XXXXXXX, KC_PSLS, KC_PAST, KC_PMNS, KC_PEQL, XXXXXXX, _______, _______,
+		XXXXXXX, XXXXXXX, KC_MPRV, KC_MPLY, KC_MNXT, XXXXXXX, XXXXXXX, KC_P7,   KC_P8,   KC_P9,   KC_PPLS, XXXXXXX, XXXXXXX, RESET,
+		_______, XXXXXXX, KC_MUTE, KC_VOLD, KC_VOLU, XXXXXXX, XXXXXXX, KC_P4,   KC_P5,   KC_P6,   KC_PPLS, XXXXXXX, _______,
+		_______, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, KC_P1,   KC_P2,   KC_P3,   KC_PENT, XXXXXXX, XXXXXXX, XXXXXXX,
+		_______, _______, _______, _______, _______, KC_P0,   KC_PDOT, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX
+		)
+};
```
Comment (drashna) [inline keyboards/dz60/keymaps/spotpuff/keymap.c:80]:
```suggestion
    [_MEDIA] = LAYOUT_directional(
        XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, KC_NLCK, XXXXXXX, KC_PSLS, KC_PAST, KC_PMNS, KC_PEQL, XXXXXXX, _______, _______,
        XXXXXXX, XXXXXXX, KC_MPRV, KC_MPLY, KC_MNXT, XXXXXXX, XXXXXXX, KC_P7,   KC_P8,   KC_P9,   KC_PPLS, XXXXXXX, XXXXXXX, RESET,
        _______, XXXXXXX, KC_MUTE, KC_VOLD, KC_VOLU, XXXXXXX, XXXXXXX, KC_P4,   KC_P5,   KC_P6,   KC_PPLS, XXXXXXX, _______,
        _______, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, KC_P1,   KC_P2,   KC_P3,   KC_PENT, XXXXXXX, XXXXXXX, XXXXXXX,
        _______, _______, _______, _______, _______, KC_P0,   KC_PDOT, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX
        )
};
```

**Review by drashna — CHANGES_REQUESTED** [review]
And please revert the changes to the VUSB submodule.

**Comment by spotpuff** [comment]
> And please revert the changes to the VUSB submodule.

OK; I reverted dd950f9eb3bad365961d2aa6c97c9dfbdd6219db and staged/pushed back. Hope that does it.

**Review by noroadsleft — CHANGES_REQUESTED** [review]
@spotpuff,

From your `qmk_firmware` directory, run:

```shell
cd lib/vusb/
git checkout bdb53e4c043d089279d9891b68bea77614cb97ee
cd ../../
git add -- lib/vusb/
git commit -m "fix vusb submodule"
git push
```

**Comment by spotpuff** [comment]
> @spotpuff,
> 
> From your `qmk_firmware` directory, run:
> 
> ```shell
> cd lib/vusb/
> git checkout bdb53e4c043d089279d9891b68bea77614cb97ee
> cd ../../
> git add -- lib/vusb/
> git commit -m "fix vusb submodule"
> git push
> ```

OK done on the few branches I had.

File: keyboards/dz60/keymaps/coppertop/rules.mk [hunk 9]
```diff
@@ -0,0 +1 @@
+BOOTMAGIC_ENABLE = no	# Virtual DIP switch configuration(+1000)
```
Comment (fauxpark) [inline keyboards/dz60/keymaps/coppertop/rules.mk:1]:
```suggestion
BOOTMAGIC_ENABLE = no
```

File: keyboards/dz60/keymaps/spotpuff/rules.mk [hunk 10]
```diff
@@ -0,0 +1 @@
+BOOTMAGIC_ENABLE = no	# Virtual DIP switch configuration(+1000)
```
Comment (fauxpark) [inline keyboards/dz60/keymaps/spotpuff/rules.mk:1]:
```suggestion
BOOTMAGIC_ENABLE = no
```

File: users/spotpuff/rules.mk [hunk 11]
```diff
@@ -0,0 +1 @@
+BOOTMAGIC_ENABLE = no	# Virtual DIP switch configuration(+1000)
```
Comment (fauxpark) [inline users/spotpuff/rules.mk:1]:
```suggestion
BOOTMAGIC_ENABLE = no
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

File: keyboards/dz60/keymaps/coppertop/keymap.c [hunk 12]
```diff
@@ -0,0 +1,80 @@
+#include QMK_KEYBOARD_H
```
Comment (noroadsleft) [inline keyboards/dz60/keymaps/coppertop/keymap.c:1]: Could you add a license header here, please?

File: keyboards/dz60/keymaps/spotpuff/keymap.c [hunk 13]
```diff
@@ -0,0 +1,80 @@
+#include QMK_KEYBOARD_H
```
Comment (noroadsleft) [inline keyboards/dz60/keymaps/spotpuff/keymap.c:1]: And here also?

**Review by noroadsleft — CHANGES_REQUESTED** [review]

File: keyboards/dz60/keymaps/coppertop/keymap.c (hunk 12 above)
Comment (spotpuff) [reply keyboards/dz60/keymaps/coppertop/keymap.c:1]: I can but I need an example; the base map I based my keymap off of had the same first line I had.

Comment (noroadsleft) [reply keyboards/dz60/keymaps/coppertop/keymap.c:1]: https://github.com/qmk/qmk_firmware/blob/410d09675a58f1890c24068066bb8eace21c2416/quantum/template/base/keymaps/default/keymap.c#L1-L15

**Review by spotpuff — COMMENTED** [review]
Added GNU license copyright text.

File: keyboards/dz60/keymaps/coppertop/keymap.c (hunk 12 above)
Comment (spotpuff) [reply keyboards/dz60/keymaps/coppertop/keymap.c:16]: OK; added to both keymaps.

File: keyboards/dz60/keymaps/spotpuff/keymap.c (hunk 13 above)
Comment (spotpuff) [reply keyboards/dz60/keymaps/spotpuff/keymap.c:16]: Done

File: keyboards/dz60/keymaps/coppertop/keymap.c (hunk 12 above)
Comment (spotpuff) [reply keyboards/dz60/keymaps/coppertop/keymap.c:16]: Done

**Review by noroadsleft — APPROVED** [review]

**Comment by noroadsleft** [comment]
For future reference, we recommend against committing to your `master` branch as you've done here, because pull requests from modified `master` branches can make it more difficult to keep your QMK fork updated. It is highly recommended for QMK development – regardless of what is being done or where – to keep your master updated, but **NEVER** commit to it. Instead, do all your changes in a branch (branches are basically free in Git) and issue PRs from your branches when you're developing.

There are instructions on how to keep your fork updated here:

[**Best Practices: Your Fork's Master: Update Often, Commit Never**](https://docs.qmk.fm/#/newbs_git_using_your_master_branch)

[Fixing Your Branch](https://docs.qmk.fm/#/newbs_git_resynchronize_a_branch) will walk you through fixing up your `master` branch moving forward. If you need any help with this just ask.

Thanks for contributing!

**Comment by spotpuff** [comment]
> For future reference, we recommend against committing to your `master` branch as you've done here, because pull requests from modified `master` branches can make it more difficult to keep your QMK fork updated. It is highly recommended for QMK development – regardless of what is being done or where – to keep your master updated, but **NEVER** commit to it. Instead, do all your changes in a branch (branches are basically free in Git) and issue PRs from your branches when you're developing.
> 
> There are instructions on how to keep your fork updated here:
> 
> [**Best Practices: Your Fork's Master: Update Often, Commit Never**](https://docs.qmk.fm/#/newbs_git_using_your_master_branch)
> 
> [Fixing Your Branch](https://docs.qmk.fm/#/newbs_git_resynchronize_a_branch) will walk you through fixing up your `master` branch moving forward. If you need any help with this just ask.
> 
> Thanks for contributing!

OK thanks for the advice. I am new to Git and it's a lot to take in, but those guides help.

**Comment by noroadsleft** [comment]
> OK thanks for the advice. I am new to Git and it's a lot to take in, but those guides help.

My favorite Git teaching resource: https://learngitbranching.js.org/

**Pull request merged**